### PR TITLE
Added webhook execute request

### PR DESCRIPTION
**Pull request by TheNumberOne**
This adds the WebhookExecuteRequest necessary for implementing executing Webhooks

### CODE REVIEW

File: src/main/java/discord4j/discordjson/json/WebhookExecuteRequest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+public interface WebhookExecuteRequest {
+
+    static ImmutableWebhookExecuteRequest.Builder builder() {
+        return ImmutableWebhookExecuteRequest.builder();
+    }
+
+    Possible<String> content();
+    Possible<String> username();
+    @JsonProperty("avatar_url")
+    Possible<String> avatar_url();
```
Comment (darichey) [inline src/main/java/discord4j/discordjson/json/WebhookExecuteRequest.java:23]: Shouldn't this be avatarUrl?

Comment (TheNumberOne) [reply src/main/java/discord4j/discordjson/json/WebhookExecuteRequest.java:23]: Oof, you are right. I will get a pull request for that asap XD

Comment (quanticc) [reply src/main/java/discord4j/discordjson/json/WebhookExecuteRequest.java:23]: Fixed

Comment (TheNumberOne) [reply src/main/java/discord4j/discordjson/json/WebhookExecuteRequest.java:23]: thanks

**Pull request merged**